### DISCUSSION TITLE
Broken link to Scanning Coverage in README.md

### DISCUSSION THREAD

**Discussion by antoson** [aquasecurity/trivy#4711] [Documentation]
### Description

Hi :)

The page Scanning Coverage doesn't exist in docs for [latest](https://aquasecurity.github.io/trivy/latest/getting-started/coverage/), where the link in README points to. The page exists only for [dev](https://aquasecurity.github.io/trivy/dev/getting-started/coverage/) docs.

### Link

https://github.com/aquasecurity/trivy/blob/85c681d443530653a62bbcb6e62f691288c01c82/README.md?plain=1#L34

### Suggestions

Make the page Scanning Coverage exist on latest docs, or point to the page in dev docs.

**Comment by knqyf263** [comment]
Thanks. The next release will generate the page.
